app/views/finance/financial_transactions/_transactions.html.haml

```diff
@@ -16,8 +16,15 @@
       - if with_ordergroup
         %th= heading_helper FinancialTransaction, :ordergroup
       %th= heading_helper FinancialTransaction, :user
+      - if FinancialTransactionType.count > 1
+        %th= heading_helper FinancialTransaction, :financial_transaction_type
       %th= sort_link_helper heading_helper(FinancialTransaction, :note), "note"
-      %th= sort_link_helper heading_helper(FinancialTransaction, :amount), "amount"
+      - FinancialTransactionClass.table_columns.each do |c|
+        %th
+          - if c.name?
+            = sort_link_helper c.name, "amount"
```

Comment:
We could have `.name?` in the model fallback to the translation, would work on more places.